distributed/tests/test_scheduler.py

```diff
@@ -3102,13 +3102,9 @@ async def test_transition_counter(c, s, a, b):
     assert s.transition_counter > 1
 
 
-@pytest.mark.skipif(MACOS and sys.version_info < (3, 9), reason="GH#5056")
-@pytest.mark.slow
 @gen_cluster(
     client=True,
     nthreads=[("127.0.0.1", 1) for _ in range(10)],
-    # typical runtime just 2-3s but on CI this may increase significantly
-    timeout=60,
 )
```

Comment:
dacorator can now fit on 1 line

Reply:
well, keeping an additional line reduces diffs if the decorator is ever changed again. I personally also consider this to be easier readable.
The point of having an auto formatter like `black` running is to not have to argue about formatting. Since black is fine with this, I'll keep it as is